.eslintrc.js

```diff
@@ -61,5 +61,7 @@ module.exports = {
 				ignoreDeclarationSort: true,
 			},
 		],
+
+		'react/prop-types': 'off'
```

Comment:
is dit iets dat we in de toekomst terug willen aanzetten? of heeft die typecheck geen nut?

Reply:
Binnen TS geen nut https://reactjs.org/docs/typechecking-with-proptypes.html